zntrack/core/base.py

```diff
@@ -698,33 +698,35 @@ def operating_directory(
         move_on: EXCEPTION_OR_LST_EXCEPTIONS = Exception,
         disable: bool = None,
     ) -> bool:
-        """Work in an operating directory until successfully finished.
-
-        This context manager will replace $nwd$ with 'prefix_$nwd$' and move the files
-        to $nwd$ when successfully finished. This can be useful, when you are running
-        e.g., on hardware with limited execution time and can't use 'dvc checkpoints'.
-        When successfully finished, all files will be moved from 'temp_$nwd$' to $nwd$.
-        You can call 'dvc repro' multiple times to continue from 'temp_$nwd$'.
-        If used properly this will result in reproducible data but:
+        """Work in an temporary operating directory until successfully finished.
+
+        This context manager will replace $nwd$ with a temporary operating
```

Comment:
Maybe we write `replace the path to the node working directory $nwd$` so we introduce the abbreviation.